File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int ind = 1;
        for(int i = 1; i<nums.size(); i++){
            if(nums[i]>nums[i-1]){
                nums[ind] = nums[i];
                ind++;
            }
        }
        return ind;
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        ind = 1
        n = len(nums)
        for i in range(1, n):
            if nums[i] > nums[i-1]:
                nums[ind] = nums[i]
                ind += 1
        return ind
        
            